sample.docker-compose.yml

```diff
@@ -1,31 +0,0 @@
-# Docker Compose sample file for freeCodeCamp
```

Comment:
No need to have this as a sample file, devs can customise behaviour with over ride files.